xbmc/music/windows/GUIWindowMusicBase.cpp

```diff
@@ -706,7 +706,10 @@ void CGUIWindowMusicBase::UpdateButtons()
   msg2.SetLabel(g_localizeStrings.Get(744)); // Files
   g_windowManager.SendMessage(msg2);
 
-  msg2.SetLabel(g_localizeStrings.Get(15100)); // Library
+  msg2.SetLabel(g_localizeStrings.Get(14022)); // Library
```

Comment:
Because they both refer to the same menu item. It's created with label 14022, but then refreshed with label 15100. Seems just wrong.